architecture/topics/aws_getting_started_before.adoc

```diff
@@ -0,0 +1,214 @@
+////
+
+AWS Service Instance Policy::
+You must create specific credential and permissions policies in AWS for the AWS Service Broker user or role.
+
+You can create the two required policies using the xref:aws-getting-started-before-policies[provided JSON samples], then assign the policies to the AWS Service Broker user or role or to a separate role, see [Managing Resources Via Assumed Role](/docs/README.md#managing-resources-via-assumed-role).
+
+One policy assigns credentials for provisioning service instances and for broker operations, such as  
+fetching the catalog and reading and writing metadata to DynamoDB.
+
+The second policy assigns permissions for provisioning, binding, and deprovisioning service instances. 
+
+Both of these policies take your AWS account ID, user ID and ARN. You can find this information in AWS. If you have the 
+AWS CLI, use the following command to get your AWS account information:
+
+[source,bash]
+----
+aws sts get-caller-identity
+{
+    "Account": "474574", 
+    "UserId": "AIDAA7ZGME4S", 
+    "Arn": "arn:aws:iam::474574:user/Administrator"
+----
+
+AWS User::
+The AWS Service Broker requires an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html[AWS Identity Access Management (IAM) user] and the associated link:http://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html[IAM keys] to provision AWS services. The user needs *Programmatic access* and *AWS Management Console* access types and add the two policies you created.
+
+IAM Role::
+The AWS Service Broker requires an IAM Role to access link:https://aws.amazon.com/cloudformation/[AWS CloudFormation] when provisioning services. 
+You must give CloudFormation full permission to the role. 
+
+Use the the AWS Management Web Console to create this role for the *CloudFormation* entity and assign *AdministratorAccess* to the role.
+
+Amazon Resource Name::
+After creating the role, locate the Amazon Resource Name (ARN) on the role access summary page using the AWS console. You need the role ARN during the deployment process.
+
+The role ARN has the following format:
+
+[source,bash]
+----
+arn:aws:iam::47458525407874:role/CloudFormation
+----
+
+[NOTE]
+====
+Optionally, the AWS Service Broker can link:https://github.com/awslabs/aws-servicebroker/tree/master/docs#managing-resources-via-assumed-role[manage multiple clusters across AWS accounts using a single role].
+====
+
+AWS Command Line Interface::
+If you want to use the command line to work with AWS, you must install the link:https://docs.aws.amazon.com/cli/latest/userguide/installing.html[AWS Command Line Interface], which requires
+link:https://developers.redhat.com/blog/2018/08/13/install-python3-rhel/[Python], and link:https://access.redhat.com/solutions/1519803[pip].
+
+DynamoDB Table::
+You need to create an link:https://console.aws.amazon.com/dynamodb/home?region=us-east-1#gettingStarted:[Amazon DynamoDB] table for persistent storage.
```

Comment:
Is there any guidance for DB sizing?

Reply:
@kalexand-rh I think the sizing is in the script as `ReadCapacityUnits` and `WriteCapacityUnits`

@jwmatthews Can you comment?